### ISSUE TITLE
"\ " causing problem with embedded \emph?

### ISSUE THREAD

**Issue by nickw2066** [zepinglee/citeproc-lua#67]
**Describe the bug**
It seems "\ " causes a problem with embedded \emph within the \cite options. See below.

**Additional information**
- TeX distribution: MacTeX 2024
- Package `citation-style-language` version: v0.5.0
- LaTeX engine: `lualatex`

**To Reproduce**
```tex
% !TEX encoding = UTF-8 Unicode
\documentclass{article}

\begin{filecontents}[overwrite, noheader]{\jobname.json}
[
    {
        "id": "Hardy2019-Gemsfr",
        "type": "book",
        "author": [
            {
                "family": "Hardy",
                "given": "H. H.",
                "suffix": "II"
            }
        ],
        "title": "Exegetical Gems from Biblical Hebrew: A Refreshing Guide to Grammar and Interpretation",
        "title-short": "Gems from Biblical Hebrew",
        "publisher-place": "Grand Rapids, MI",
        "publisher": "Baker Academic",
        "issued": {
            "date-parts": [
                [
                    "2019"
                ]
            ]
        }
    }
]
\end{filecontents}

\usepackage{citation-style-language}
\cslsetup{style = chicago-fullnote-bibliography}
\addbibresource{\jobname.json}

\begin{document}

\dots\ dalam Kej.\ 8:11 sebagai penanda keterkejutan:
``\emph{The dove returned to him
at sunset, \emph{and incredibly}, a fresh-picked olive branch
was in its mouth!}''\cite[prefix = {\dots\ dalam Kej.\ 8:11
sebagai penanda keterkejutan: ``\emph{The dove returned to him
at sunset, \emph{and incredibly}, a fresh-picked olive branch
was in its mouth!}''; lihat },
page = {194 (tekanan saya)}]{Hardy2019-Gemsfr}

\end{document}
```

**Screenshots**
Main text:
<img width="735" alt="Screenshot 2024-07-09 at 12 21 04 PM" src="https://github.com/zepinglee/citeproc-lua/assets/110715564/a3f09533-6c60-4fb8-b85a-209ad239b8d6">

Citation:
<img width="732" alt="Screenshot 2024-07-09 at 12 21 10 PM" src="https://github.com/zepinglee/citeproc-lua/assets/110715564/de3261b6-99ca-436e-bc76-e3a6c69ce7ec">

**Comment by zepinglee** [comment]
Are you referring to the italic "and incredibly" in the footnote which is expected to be upright?

**Comment by nickw2066** [comment]
Yes

**Comment by zepinglee** [comment]
Ah it turns out to be a bug in the Lua module and it's fixed in [bc67668](https://github.com/zepinglee/citeproc-lua/commit/bc676686899021eee31dfa3143666e6791def67e).
